
string[] cloud_list

---

bool[] success